db/migrate/20211001151212_update_patient_table_string_lengths.rb

```diff
@@ -0,0 +1,185 @@
+    change_column :patients, :middle_name, :string, limit: 200
+    change_column :patients, :sex, :string, limit: 16
+    change_column :patients, :gender_identity, :string, limit: 128
+    change_column :patients, :sexual_orientation, :string, limit: 64
+    change_column :patients, :ethnicity, :string, limit: 32
+    change_column :patients, :nationality, :string, limit: 200
+    change_column :patients, :primary_language, :string, limit: 64
+    change_column :patients, :secondary_language, :string, limit: 64
+    change_column :patients, :user_defined_id_statelocal, :string, limit: 200
+    change_column :patients, :user_defined_id_cdc, :string, limit: 200
+    change_column :patients, :user_defined_id_nndss, :string, limit: 200
+
+    # Address
+    change_column :patients, :address_line_1, :string, limit: 200
+    change_column :patients, :address_line_2, :string, limit: 200
+    change_column :patients, :address_city, :string, limit: 200
+    change_column :patients, :address_state, :string, limit: 200
+    change_column :patients, :address_zip, :string, limit: 200
+    change_column :patients, :address_county, :string, limit: 200
+    change_column :patients, :foreign_address_line_1, :string, limit: 200
+    change_column :patients, :foreign_address_line_2, :string, limit: 200
+    change_column :patients, :foreign_address_city, :string, limit: 200
+    change_column :patients, :foreign_address_country, :string, limit: 200
+    change_column :patients, :foreign_address_zip, :string, limit: 200
+    change_column :patients, :foreign_address_line_3, :string, limit: 200
+    change_column :patients, :foreign_address_state, :string, limit: 200
+    change_column :patients, :monitored_address_line_1, :string, limit: 200
+    change_column :patients, :monitored_address_line_2, :string, limit: 200
+    change_column :patients, :monitored_address_city, :string, limit: 200
+    change_column :patients, :monitored_address_state, :string, limit: 200
+    change_column :patients, :monitored_address_zip, :string, limit: 200
+    change_column :patients, :monitored_address_county, :string, limit: 200
+    change_column :patients, :foreign_monitored_address_line_1, :string, limit: 200
+    change_column :patients, :foreign_monitored_address_line_2, :string, limit: 200
+    change_column :patients, :foreign_monitored_address_city, :string, limit: 200
+    change_column :patients, :foreign_monitored_address_state, :string, limit: 200
+    change_column :patients, :foreign_monitored_address_zip, :string, limit: 200
+    change_column :patients, :foreign_monitored_address_county, :string, limit: 200
+
+    # Contact
+    change_column :patients, :preferred_contact_method, :string, limit: 32
+    change_column :patients, :preferred_contact_time, :string, limit: 16
+    change_column :patients, :primary_telephone, :string, limit: 16
+    change_column :patients, :primary_telephone_type, :string, limit: 16
```

Comment:
Limit on this is 10

`ValidationHelper::VALID_PATIENT_ENUMS[:primary_telephone_type].reject(&:nil?).map(&:length).max`